hyper_click_command.py

```diff
@@ -29,9 +29,8 @@ def run(self, edit):
         line_content = v.substr(line_range).strip()
         matched = self.is_valid_line(line_content)
         if matched:
-            destination_str = matched.group(1)
-            file_path = HyperClickPathResolver(
-                destination_str, v.file_name(),
+            file_path = HyperClickPathResolver(v,
+                matched, v.file_name(),
```

Comment:
since you pass `v`, you don't have to pass `v.file_name()` too

Reply:
i will make the change, i haven't notice that :+1: